akka-distributed-data/src/main/resources/reference.conf

```diff
@@ -0,0 +1,61 @@
+##############################################
+# Akka Distributed DataReference Config File #
+##############################################
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+
+#//#distributed-data
+# Settings for the DistributedData extension
+akka.cluster.distributed-data {
+  # Actor name of the Replicator actor, /system/replicator
+  name = replicator
```

Comment:
good point, I think we tend to write actor names in camel: dataReplicator (it's weird that I'm thinking about the extra bytes that it adds to the remote messages)